### ISSUE TITLE
Coda wrap with abbriviation doesn't work on PHP

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#219]
```
In Coda when wrapping abbriviation around code include PHP this happens:

Before:
<?php foreach($field_medarbejder_bullets as $bullet) : ?>
   <li><?php echo $bullet['view']; ?></li>
<?php endforeach; ?>

After:
<div class="wrap"><?php foreach(\$field_medarbejder_bullets as \$bullet) : ?>
  <li><?php echo \$bullet['view']; ?></li>
<?php endforeach; ?></div>

Notice the \ in front of every variable...
```

---

Original issue reported on code.google.com by `createin...@gmail.com` on 19 Nov 2010 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 1 Feb 2011 at 10:56
- Added labels: **Milestone-0.7**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 2 Feb 2011 at 10:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Sounds great! Thank You!
```

---

Original comment by `createin...@gmail.com` on 3 Feb 2011 at 8:13
